fastlane/Pluginfile

```diff
@@ -6,5 +6,5 @@ group :screenshots, optional: true do
   gem 'rmagick', '~> 4.1'
 end
 
-gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'trunk'
-# gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
+#gem 'fastlane-plugin-wpmreleasetoolkit', git: 'git@github.com:wordpress-mobile/release-toolkit.git', branch: 'trunk'
+gem 'fastlane-plugin-wpmreleasetoolkit', '~> 4.1'
```

Comment:
I'm afraid this was an intentional breakage of our convention of always pointing to a stable release of `trunk`.

We haven't shipped a new version of the toolkit since merging [the GlotPress 429 workaround](https://github.com/wordpress-mobile/release-toolkit/pull/361). See [the current `CHANGELOG.md`](https://github.com/wordpress-mobile/release-toolkit/blob/48bf4f8ee18079154b2054b3014bed9f114416d6/CHANGELOG.md#trunk).

Although, thinking about it... It's conceivable that we'll have a new release out before the next _beta_ (as we don't need to download from GlotPress during code freeze, and the release branch "correctly" points to `trunk` still), so maybe we can keep this as is.

(_I'm leaving a comment about making a release instead of doing it myself because I'm at EOD and I don't want to start something else at this point_)

Reply:
Addressed by releasing release-toolkit `4.2.0` earlier today (which includes both the changes from `4.1` but also the changes that WPAndroid was pointing to to get the rate limiting fix from the toolkit which were not released yet before), then updating the `Pluginfile` and re-ran `bundle install` to update the `Gemfile.lock` accordingly.

See https://github.com/wordpress-mobile/WordPress-Android/pull/16642/commits/7e3abb5b9d754a4a4f87c3a622c75b7a0a1019b6